process/block/metablock.go

```diff
@@ -1506,8 +1520,44 @@ func (mp *metaProcessor) createPeerInfo() ([]block.PeerData, error) {
 	return peerInfo, nil
 }
 
+func (mp *metaProcessor) verifyTotalAccumulatedFeesInEpoch(blockChain data.ChainHandler, metaHdr *block.MetaBlock) error {
+	computedTotalFees, err := mp.computeAccumulatedFeesInEpoch(blockChain, metaHdr)
+	if err != nil {
+		return err
+	}
+
+	if computedTotalFees.Cmp(metaHdr.AccumulatedFeesInEpoch) != 0 {
+		return process.ErrAccumulatedFeesDoNotMatch
```

Comment:
Might be a good idea to wrap here the error, something like:
```GO
return fmt.Errorf("%w, got %v, computed %v", process.ErrAccumulatedFeesDoNotMatch, metaHdr.AccumulatedFeesInEpoch, computedTotalFees)
```

Reply:
done.